package.yaml

```diff
@@ -47,6 +47,7 @@ library:
     - fast-logger
     - filepath
     - formatting
+    - generic-lens
```

Comment:
yay for `generic-lens` 😉

Reply:
Isn't `view (the @"foo")` cute?

(Between the two of us: it does break all module export restrictions (_sic_), though :no_mouth:)